Makefile

```diff
@@ -92,3 +92,8 @@ doctest:
 	$(SPHINXBUILD) -b doctest $(ALLSPHINXOPTS) $(BUILDDIR)/doctest
 	@echo "Testing of doctests in the sources finished, look at the " \
 	      "results in $(BUILDDIR)/doctest/output.txt."
+
+pdf:
+	$(SPHINXBUILD) -b pdf $(ALLSPHINXOPTS) _build/pdf
+	@echo
+	@echo "Build finished. The PDF Version of Documentation is in _build/pdf."
```

Comment:
Sphinx uses pdfLaTeX to generate PDF documentation, and Read the Docs uses TeX Live in its build infrastructure. PDF output with rst2pdf is by far not as good as when using LaTeX but it has the real advantage of not requiring you to install a full LaTeX stack on your machine.

The package to install for Latex based pdf generation is "texlive-full"